src/search/searchBox/SearchBox.js

```diff
@@ -29,6 +29,7 @@ class SearchBox extends React.Component {
     };
 
     render() {
+        const searchCount = this.props.searchResult ? this.props.searchResult.total : '...';
```

Comment:
`searchCount ` skal vel være antall annonser totalt? At det skal stå feks "Søk i 13450 ledige annonser" også når man har trykket på ett eller flere filtere. Her vil vel searchCount tilsvare antall treff på et spesifikt søk, og at teksten i verste fall kan bli "Søk i 0 ledige annonser"?

Reply:
Ja jeg tenkte at det var best å la antallet gjenspeile antall stillinger man kan treffe med aktive filtere. Om man har aktive filtere vil jo ikke brukere kunne søke blant absolutt alle stillinger om man skriver noe inn i søkeboksen og trykker søk, så syntes selv det var litt misvisende å la den vise totalt antall stillinger.

Reply:
Dette er kanskje et spørsmål for UX?

Reply:
Har snakket med Malin. Vi dropper tallet siden ingen av de to alternativene er helt gode.